charts/hono/example/create_certs.sh

```diff
@@ -105,7 +105,7 @@ openssl req -config ca_opts -reqexts intermediate_ext -new -key $DIR/ca-key.pem
 
 echo ""
 echo "creating PEM trust store ($DIR/trusted-certs.pem) containing CA certificate"
-cat $DIR/ca-cert.pem $DIR/root-cert.pem > $DIR/trusted-certs.pem
+cat $DIR/ca-cert.pem $DIR/root-cert.pem lets-encrypt-root-cert.pem > $DIR/trusted-certs.pem
```

Comment:
maybe we can download the cert from the Let's Encrypt site instead? That would relief us from having to maintain a copy in the repo ...